docs/site/docusaurus.config.js

```diff
@@ -50,7 +50,51 @@ const config = {
 
     plugins: [
         "docusaurus-plugin-copy-page-button",
-            [
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        fromExtensions: ['html', 'htm'], // /myPage.html -> /myPage
+        redirects: [
+{
+            to: "/docs/design/encoding",
+            from: "/design/encoding.html",
+          },
+          {
+            to: "/docs/usage/client-cli",
+            from: "/usage/client-cli.html",
+          },
+          {
+            to: "/docs/usage/interacting",
+            from: "/usage/interacting.html",
+          },
+          {
+            to: "/docs/usage/quilt",
+            from: "/usage/quilt.html",
+          },
+          {
+            to: "/docs/usage/setup",
+            from: "/usage/setup.html",
+          },
+          {
+            to: "/docs/usage/web-api",
+            from: "/usage/web-api.html",
+          },
+          {
+            to: "/docs/walrus-sites/bring-your-own-domain",
+            from: "/walrus-sites/bring-your-own-domain.html",
+          },
+          {
+            to: "/docs/walrus-sites/intro",
+            from: "/walrus-sites/intro.html",
+          },
+          {
+            to: "/docs/walrus-sites/portal",
+            from: "/walrus-sites/portal.html",
+          },
+        ],
```

Comment:
Question: How did you select the pages that need to be redirected? For more complete coverage, would it be possible to have a wild-card redirect from `/walrus-sites/` to `/docs/walrus-sites/`, from `/usage/` to `/docs/usage/`, etc.?